modules/apps/headless-apio/media-object/media-object-apio-test/src/testIntegration/java/com/liferay/media/object/apio/internal/architect/resource/test/DefaultMediaObjectNestedCollectionResourceTest.java

```diff
@@ -213,6 +299,43 @@ private String _getFileEntryPreviewURL(FileEntry fileEntry)
 			getNestedCollectionResource(), fileEntry);
 	}
 
+	private void _verifyAdaptiveMedia(
```

Comment:
we should call this method assert, could we create a Matcher instead to use a generic method?